src/Language/PureScript/AST/Declarations.hs

```diff
@@ -260,6 +260,33 @@ instance Eq DeclarationRef where
   r == (PositionedDeclarationRef _ _ r') = r == r'
   _ == _ = False
 
+-- enable sorting lists of explicitly imported refs
+instance Ord DeclarationRef where
+  compare = compDecRef
+
+compDecRef :: DeclarationRef -> DeclarationRef -> Ordering
+compDecRef (TypeRef name _) (TypeRef name' _) = compare name name'
+compDecRef (TypeOpRef name) (TypeOpRef name') = compare name name'
+compDecRef (ValueRef ident) (ValueRef ident') = compare ident ident'
+compDecRef (ValueOpRef name) (ValueOpRef name') = compare name name'
+compDecRef (TypeClassRef name) (TypeClassRef name') = compare name name'
+compDecRef (TypeInstanceRef ident) (TypeInstanceRef ident') = compare ident ident'
+compDecRef (ModuleRef name) (ModuleRef name') = compare name name'
+compDecRef (ReExportRef name _) (ReExportRef name' _) = compare name name'
+compDecRef (PositionedDeclarationRef _ _ ref) (PositionedDeclarationRef _ _ ref') = compare ref ref'
```

Comment:
Actually on second thought this breaks consistency with regards to the `Eq` instance so maybe it's better to not make this an `Ord` instance and just use it as a function with `sortWith`